### PR TITLE
Documentation : Improve doc for the CLI (3) #551

### PR DESCRIPTION
**Pull request by cserf**
Documentation : Improve doc for the CLI (3) #551

### CODE REVIEW

File: bin/rucio [hunk 1]
```diff
@@ -358,16 +359,9 @@ def list_file_replicas(args):
 
     table = []
     dids = []
-    rse_dict = {}
     if args.missing and not args.selected_rse:
         print 'Cannot use --missing without specifying a RSE'
         return FAILURE
-    if args.selected_rse and args.list_collections:
```
Comment (mlassnig) [inline bin/rucio:365]: are these removals on purpose?

Comment (cserf) [reply bin/rucio:365]: Yes, the option was deprecated for a long time :
    list_file_replicas_parser.add_argument('--list-collections', dest='list_collections', action='store_true', default=False, help='To have the number of files of the dataset by site\
        (' + Color.BOLD + 'DEPRECATED: ' + Color.END + 'please use list-dataset-replicas instead.)', required=False)

File: bin/rucio [hunk 2]
```diff
@@ -524,6 +498,7 @@ def detach(args):
         cscope, cname = extract_scope(did)
         dids.append({'scope': cscope, 'name': cname})
     client.detach_dids(scope=scope, name=name, dids=dids)
+    print 'DIDs successfully detached from %s:%s' % (scope, name)
```
Comment (bari12) [inline bin/rucio:501]: I am not for or against this, but we are not very consistent with these messages in general. For some commands we follow the "bash" mantra that we do not give a message on a successful command. On other we do give a message. I think we should be consistent here.

Comment (cserf) [reply bin/rucio:501]: The idea here was to be consistent with the attach command:

    client.attach_dids(scope=scope, name=name, dids=dids)
    print 'DIDs successfully attached to %s:%s' % (scope, name)

I agree we need to be consistent on every commands. We will need to work on this after the workshop.

**Review by bari12 — APPROVED** [review]

**Pull request merged**